index.html

```diff
@@ -1,20 +1,268 @@
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div
+            class="collapse navbar-collapse justify-content-center text-center"
+            id="navbarNavAltMarkup"
+          >
+            <div class="navbar-nav">
+              <a class="nav-link active" aria-current="page" href="index.html"
+                >About</a
+              >
+              <a class="nav-link" href="resume.html">Resume</a>
+              <a class="nav-link" href="portfolio.html">Portfolio</a>
+              <a class="nav-link" href="contact.html">Contact</a>
+            </div>
+          </div>
+        </div>
+      </nav>
+    </header>
 
+    <main>
+      <div class="container mt-5">
+        <div class="row">
+          <div class="col-md-offset-4 col-md-5">
+            <!-- Business card-->
+            <div class="container business-card shadow p-sm-3">
+              <div class="row">
+                <div
+                  class="col-md-5 d-flex justify-content-center"
+                  style="width: 50%; margin: auto"
+                >
+                  <img
+                    src="img/profile-me.png"
+                    class="profile-pic"
+                    alt="Profile picture"
+                  />
+                </div>
+              </div>
+              <div class="row">
+                <div
+                  class="col-md-5 justify-content-center"
+                  style="width: 50%; margin: auto"
+                >
+                  <h3 class="card-title text-wrap text-center">
+                    Laura Menendez
+                  </h3>
+                </div>
+              </div>
+              <div class="row">
+                <div class="col-md-5 text-center mx-auto">
+                  <h2 class="line">&#8212;</h2>
+                </div>
+              </div>
+              <div class="row">
+                <div class="col-md-5" style="width: 60%; margin: auto">
+                  <h5 class="text-nowrap" style="line-height: 1.5em">
+                    TRAINEE DEVELOPER
+                  </h5>
+                </div>
+              </div>
+              <!-- Extra resume and portfolio buttons -->
+              <!-- div class="row">
+              <div
+                class="col-md-5 d-sm-inline d-md-inline"
+                style="width: 60%; margin: auto"
+              >
+                <a href="resume.html" class="btn resume">RESUME</a>
+              </div>
+              <div
+                class="col-md-5 d-sm-inline d-md-inline"
+                style="width: 40%; margin: auto"
+              >
+                <a
+                  href="portfolio.html"
+                  class="btn btn-outline-primary portfolio"
+                  >PORTFOLIO</a
+                >
+              </div>
+            </div-->
+              <div class="row">
+                <div class="bg-white icons">
+                  <i class="fa fa-facebook-f" style="font-size: 24px"></i>
```

Comment:
these icons look very neat! Make sure you use the available sizing CSS class for FontAwesome, for example ``fa-xs`` or ``fa-sm``, as mentioned in https://fontawesome.com/how-to-use/on-the-web/styling/sizing-icons